target/scripts/build/packages.sh

```diff
@@ -94,26 +94,12 @@ function _install_dovecot() {
   declare -a DOVECOT_PACKAGES
 
   DOVECOT_PACKAGES=(
-    dovecot-core dovecot-fts-xapian dovecot-imapd
+    dovecot-core dovecot-imapd dovecot-dev
```

Comment:
We are always using a compiled FTS Xapian now? How much does that add to build time :thinking: (_I suppose it's fine, cache should be fine so long as it's not invalidated by this `packages.sh` file being updated too often_ :sweat_smile: )

Reply:
Currently it would always use the compiled one. Did not measure the time, but packages.sh seems to be most of the build time.